okd/scripts/run.sh

```diff
@@ -54,17 +54,11 @@ do
     sleep 5
 done
 
-# TODO: do not sure if it is better way to check that the whole cluster is up, under the cluster we should have
-# only one router pod in the pending state, because we have only one worker node, but router deployment wants to have
-# two replicas
-until [[ $(oc get pods --all-namespaces --no-headers | grep -v Running | grep -v Completed | wc -l) == 1 ]]; do
-    sleep 10
-done
-
 # wait minute, just to be sure we do not get old cluster state
-sleep 60
+sleep 30
 
-until [[ $(oc get pods --all-namespaces --no-headers | grep -v Running | grep -v Completed | wc -l) == 1 ]]; do
+until [[ $(oc get pods --all-namespaces --no-headers | grep -v Running | grep -v Completed | wc -l) -gt 3 ]]; do
```

Comment:
Can you explain this line

Reply:
it's waiting for all pods to be either in a RUNNING or COMPLETED state.

the '3' here is to account for 3 pods that won't come online regardless of what we do because we're only running 2 nodes. They are deployments with replica counts > 2 that can't be scheduled.

Reply:
Thanks for the comment!